azure-pipelines.yml

```diff
@@ -1120,7 +1124,156 @@ jobs:
+      displayName: 'Testing git pull and version'
+          
+    # - bash: |
+    #     rm -rf bin/qt.zip
+    #     rm -rf bin/5.12.1
+   
+    #     df -k .
+    #   displayName: 'Disk space check'
+
+    # # Configure and build CaPTk
+    # - bash: |
+    #     cd bin
+    #     export CMAKE_PREFIX_PATH=$(Build.SourcesDirectory)/bin/ITK-build:$(Build.SourcesDirectory)/bin/DCMTK-build:$(Build.SourcesDirectory)/bin/qt/5.12.1/lib/cmake/Qt5:$CMAKE_PREFIX_PATH
+    #     cmake -DCMAKE_INSTALL_PREFIX="./install/appdir/usr" -DITK_DIR=./ITK-build -DDCMTK_DIR=./DCMTK-build -DBUILD_TESTING=ON ..
+    #     make -j2
+    #   displayName: 'Configure and compile CaPTk with Testing enabled'
+
+    # - script: |
+    #    cd bin
+    #    make test
+    #    rm -rf testing
+    #   displayName: 'Performing the Tests'
+
+    # Configure and build CaPTk
+    - bash: |
+        cd bin
+        # export PATH=$(Build.SourcesDirectory)/bin/qt/5.12.1/bin:$(Build.SourcesDirectory)/bin/qt/5.12.1/libexec:$PATH
+        # export CMAKE_PREFIX_PATH=$(Build.SourcesDirectory)/bin/ITK-build:$(Build.SourcesDirectory)/bin/DCMTK-build:$(Build.SourcesDirectory)/bin/qt/5.12.1/lib/cmake/Qt5:$CMAKE_PREFIX_PATH
+        
+        cmake -DCMAKE_INSTALL_PREFIX="./install/appdir/usr" -DBUILD_TESTING=OFF ..
+        
+        rm -rf binaries_linux.zip
+        rm -rf qt.zip
+        rm -rf 5.12.1
+
+        rm -rf externalApps/binaries_linux/libra
+
+        make -j2
+        make install/strip
+      displayName: 'Configure and compile CaPTk'
+
+    # - bash: |
+    #     rm -rf bin/qt.zip
+    #     rm -rf bin/5.12.1
+    #     rm -rf regression
+    #     rm -rf binaries
+    #     rm -rf bin_linux_*.tar.gz
+    #     df -k .
+    #   displayName: 'Disk space check'
+
+    # Build a package
+    - bash: |
+        # export PATH=$(Build.SourcesDirectory)/bin/qt/5.12.1/bin:$(Build.SourcesDirectory)/bin/qt/5.12.1/libexec:$PATH
+        # export CMAKE_PREFIX_PATH=$(Build.SourcesDirectory)/bin/ITK-build:$(Build.SourcesDirectory)/bin/DCMTK-build:$(Build.SourcesDirectory)/bin/qt/5.12.1/lib/cmake/Qt5:$CMAKE_PREFIX_PATH
+
+        # rm -rf regression
+        # rm -rf binaries
+
+        export PKG_FAST_MODE=1
+        export ARCH=x86_64 appimagetool
+
+        # export PKG_COPY_QT_LIBS=1
+        ./scripts/captk-pkg
+
+        rm -rf src
+        rm -rf docs*
+
+        rm -rf scripts
+
+        mv *_Installer.* $(Build.ArtifactStagingDirectory)
+
+        rm -rf bin
+        df -k .
+
+      displayName: 'Set up environment variables for packager and build a package'
+
+    - task: PublishBuildArtifacts@1
+      inputs:
+        pathtoPublish: '$(Build.ArtifactStagingDirectory)'
+        artifactName: '$(Build.DefinitionName)_installer_Linux_ubuntu1604'
+        continueOnError: true
+
+    - task: DownloadBuildArtifacts@0
+      inputs:
+        buildType: 'current'
+        downloadType: 'single'
+        artifactName: '$(Build.DefinitionName)_installer_Linux_ubuntu1604'
```

Comment:
These two artifacts should be labelled as centos7 rather than ubuntu.

Reply:
Could the installers be named as ```$(Build.DefinitionName)_Installer_centos7```?

Reply:
My bad. Will add that in.